tests/defaultCleanOptions.tests.js

```diff
@@ -0,0 +1,27 @@
+import expect from 'expect';
+import Collection2 from 'meteor/aldeed:collection2';
+
+describe('defaultCleanOptions', function () {
```

Comment:
You've renamed it from ~`defaultCleanOptions`~ to `cleanOptions`, but maybe this was kept on purpose?